app/presenters/get_involved_presenter.rb

```diff
@@ -0,0 +1,59 @@
+class GetInvolvedPresenter
+  attr_reader :content_item
+
+  def initialize(content_item)
+    @content_item = content_item
+  end
+
+  def recently_opened
+    filtered_links(content_item.recently_opened_consultations, I18n.t("formats.get_involved.closes"))
+  end
+
+  def recent_outcomes
+    filtered_links(content_item.recent_consultation_outcomes, I18n.t("formats.get_involved.closed"))
+  end
+
+  def time_until_next_closure
+    days_left = (content_item.next_closing_consultation["end_date"].to_date - Time.zone.now.to_date).to_i
+    case days_left
+    when :negative?.to_proc
+      I18n.t("formats.get_involved.closed")
+    when :zero?.to_proc
+      I18n.t("formats.get_involved.closing_today")
+    when 1
```

Comment:
This case statement is pretty hideous. I know you've copied it verbatim from government-frontend, which was the correct thing to do, but perhaps it could be refactored in another commit to use "if/else"?

i.e.

```ruby
if days_left.negative?
  ...
elsif days_left.zero?
  ...
elsif days_left == 1
 ...
else
 ...
end
```